detox/test/e2e/19.crash-handling.test.js

```diff
@@ -18,14 +18,27 @@ describe('Crash Handling', () => {
   it('Should throw error upon internal app crash', async () => {
     await device.reloadReactNative();
     await expectToThrow(() => element(by.text('Crash')).tap(), 'The app has crashed');
-    await expectToThrow(() => element(by.text('Crash')).tap(), 'Detox can\'t seem to connect to the test app(s)!');
```

Comment:
Oh wow. Feels weird now that it was ever tested this way.

Reply:
Actually I'm satisfied to see this :) This means that here we had to re-evaluate our product decision based on the real feedback.